usl_pipeline/cloud_functions/main.py

```diff
@@ -220,6 +220,42 @@ def _get_crs_from_wps(nc_dataset: netCDF4.Dataset) -> str:
     return crs
 
 
+@functions_framework.cloud_event
+@_retry_and_report_errors()
+def write_heat_scenario_config_metadata(
+    cloud_event: functions_framework.CloudEvent,
+) -> None:
+    """Writes metadata and features for uploaded heat scenario config.
+
+    This function is triggered when files for heat configuration are uploaded to
+    GCS. It writes an entry to the metastore for the configuration.
+    """
+    file_name = pathlib.PurePosixPath(cloud_event.data["name"])
+    # For AtmoML, use `Heat_Data_` files to trigger HeatScenarioConfig write
+    # to metastore
+    if re.search(file_names.HEAT_CONFIG_TXT_REGEX, file_name.name):
+        storage_client = storage.Client()
+        db = firestore.Client()
+
+        config_blob = storage_client.bucket(cloud_event.data["bucket"]).blob(
+            cloud_event.data["name"]
+        )
+        with config_blob.open("rt") as txt_fd:
+            config_dict = config_readers.read_key_value_pairs(txt_fd)
+
+        metastore.HeatScenarioConfig(
+            name=config_blob.name,
+            # File names should be in the form:
+            # <study_area>/<parent_config_name>/<file_name>
+            # Ex: NYC_Heat/Summer_Config/Heat_Data_2012.txt
+            parent_config_name=file_name.parent.name,
+            gcs_uri=f"gs://{config_blob.bucket.name}/{config_blob.name}",
+            simulation_year=int(config_dict["simulation_year"]),
+            simulation_months=config_dict["simulation_months"],
+            percentile=int(config_dict["percentile"]),
```

Comment:
is the file format for this documented? folks might expect this to be a float `.99` rather than an int `99` to express 99th percentile. I don't feel strongly either way -- just want to make sure it's written down because it is ambiguous.

Reply:
not yet! working on one that will provide example of file format